benchmark/url/url-searchparams-sort.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const common = require('../common.js');
+const URLSearchParams = require('url').URLSearchParams;
+
+const inputs = {
+  empty: '',
+  sorted: 'a&b&c&d&e&f&g&h&i&j&k&l&m&n&o&p&q&r&s&t&u&v&w&x&y&z',
+  almostsorted: 'a&b&c&d&e&f&g&i&h&j&k&l&m&n&o&p&q&r&s&t&u&w&v&x&y&z',
+  reversed: 'z&y&x&w&v&u&t&s&r&q&p&o&n&m&l&k&j&i&h&g&f&e&d&c&b&a',
+  random: 'm&t&d&c&z&v&a&n&p&y&u&o&h&l&f&j&e&q&b&i&s&x&k&w&r&g',
+  // 8 elements
+  short: 'm&t&d&c&z&v&a&n',
+  // 88 elements
+  long: 'g&r&t&h&s&r&d&w&b&n&h&k&x&m&k&h&o&e&x&c&c&g&e&b&p&p&s&n&j&b&y&z&' +
```

Comment:
Right, but querystring.parse returns something like `{ g: [ '', '' ] }` in this case, and this object is converted to `[ 'g', '', 'g', '' ]`. Either way, I added a small ad-hoc parser to skip `querystring.parse`, and to also maintain the order of the input.